requirements_all.txt

```diff
@@ -1,5 +1,4 @@
 # Home Assistant Core, full dependency set
--c homeassistant/package_constraints.txt
```

Comment:
We did this on purpose. If one ends up installing from just this file, we should maintain the constraints.

Reply:
I thought this at first too, but then i saw `requirements.txt` has the constraints as well so won't they still be applied? Or does pip not handle that properly?

Reply:
It does, they really are redundant.

```
$ cat reqs1.txt 
-r reqs2.txt
$ cat reqs2.txt 
-c cons.txt
enum34
$ cat cons.txt 
enum34==10000
$ pip install -r reqs1.txt 
Collecting enum34==10000 (from -c cons.txt (line 1))
  ERROR: Could not find a version that satisfies the requirement enum34==10000 (from -c cons.txt (line 1)) (from versions: 0.9, 0.9.1, 0.9.11, 0.9.12, 0.9.13, 0.9.14, 0.9.15, 0.9.16, 0.9.17, 0.9.18, 0.9.19, 0.9.20, 0.9.21, 0.9.22, 0.9.23, 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, 1.1.8, 1.1.9, 1.1.10)
ERROR: No matching distribution found for enum34==10000 (from -c cons.txt (line 1))
```